packages/server/api/src/app/ee/helper/email/email-service.ts

```diff
@@ -56,11 +57,12 @@ export const emailService = {
         if (EDITION_IS_NOT_PAID) {
             return
         }
+        const createdAtFormatted = dayjs(createdAt.replace(' UTC', '')).tz('PT').format('DD MMM YYYY, HH:mm [PT]')
```

Comment:
const createdAtFormatted = dayjs(createdAt).tz('PT').format('DD MMM YYYY, HH:mm [PT]')

Why not this?